composer.json

```diff
@@ -12,13 +12,14 @@
         }
     ],
     "require": {
-        "php": ">=5.4",
-        "symfony/symfony": "~2.3",
-        "symfony/framework-bundle": "~2.3",
-        "jms/payment-core-bundle": "~1.0"
+        "php": ">=7.0",
+        "ext-intl": "*",
+        "symfony/symfony": "^2.8 || ^3.0",
+        "symfony/framework-bundle": "^2.8 || ^3.0",
+        "jms/payment-core-bundle": "^1.4"
     },
     "require-dev": {
-        "phpunit/phpunit": "4.8.*"
+        "phpunit/phpunit": "^6.5"
```

Comment:
it should be phpunit 5 minimum, otherwise it will not install on our app (phpunit 5)